test/e2e/flask/multichain-api/create-session.spec.ts

```diff
@@ -261,4 +261,144 @@ describe('Multichain API', function () {
+          ...DEFAULT_MULTICHAIN_TEST_DAPP_FIXTURE_OPTIONS,
+        },
+        async ({
+          driver,
+          extensionId,
+        }: {
+          driver: Driver;
+          extensionId: string;
+        }) => {
+          await openMultichainDappAndConnectWalletWithExternallyConnectable(
+            driver,
+            extensionId,
+          );
+
+          await initCreateSessionScopes(driver, ['eip155:1']);
+
+          const editButtons = await driver.findElements('[data-testid="edit"]');
+          await editButtons[0].click();
+
+          const checkboxes = await driver.findElements(
+            'input[type="checkbox" i]',
+          );
+          const accountCheckbox = checkboxes[1];
+          const isChecked = await accountCheckbox.isSelected();
+
+          assert.strictEqual(
+            isChecked,
+            true,
+            'first account should be automatically selected',
+          );
+        },
+      );
+    });
+  });
+
+  describe('Connect wallet to the multichain dapp via `externally_connectable`, call `wallet_createSession`, choose to edit accounts and', function () {
+    describe('add a new one', function () {
+      it('dApp should receive a response that includes permissions for the accounts that were selected for sharing', async function () {
+        await withFixtures(
+          {
+            title: this.test?.fullTitle(),
+            fixtures: new FixtureBuilder()
+              .withPopularNetworks()
+              .withPreferencesControllerAdditionalAccountIdentities()
+              .build(),
+            ...DEFAULT_MULTICHAIN_TEST_DAPP_FIXTURE_OPTIONS,
+          },
+          async ({
+            driver,
+            extensionId,
+          }: {
+            driver: Driver;
+            extensionId: string;
+          }) => {
+            await openMultichainDappAndConnectWalletWithExternallyConnectable(
+              driver,
+              extensionId,
+            );
+
+            await initCreateSessionScopes(driver, ['eip155:1']);
+
+            await addAccountInWalletAndAuthorize(driver);
+
+            await driver.clickElement({ text: 'Connect', tag: 'button' });
+            await driver.switchToWindowWithTitle(
+              WINDOW_TITLES.MultichainTestDApp,
+            );
+
+            const getSessionScopesResult = await getSessionScopes(driver);
+
+            assert.deepEqual(
+              getSessionScopesResult.sessionScopes['eip155:1'].accounts,
+              getExpectedSessionScope('eip155:1', [
+                DEFAULT_FIXTURE_ACCOUNT,
+                SECOND_INJECTED_ACCOUNT,
+              ]).accounts,
+              'The dapp should receive a response that includes permissions for the accounts that were selected for sharing',
+            );
+          },
+        );
+      });
+    });
+
+    describe('deselect all', function () {
+      it('should not be able to update the account permissions (at least one account is required)', async function () {
```

Comment:
```suggestion
      it('should not be able to approve the create session request without any account selected', async function () {
```

Reply:
resolved in [6e0626b](https://github.com/MetaMask/metamask-extension/pull/29400/commits/6e0626b85a265db8e150209a2ae91b1a1f18ee04)